MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include versioneer.py
+include amostra/_version.py
```

Comment:
ok - a lot of those were added as the code was being upgraded for version 1.0.

Reply:
sorry about previous comment - I thought you were saying the files were not present - but you mean in manifest. Will do that and add to pull request

Reply:
I noticed that in that v0.2.x branch we have a differently named file `README.md`, not `README.rst`. Here are the files that are currently there:
![image](https://user-images.githubusercontent.com/13209176/150362115-9734d6f1-de5c-4913-b859-d4deb61627ae.png)

The `requirements.txt` file is added by this PR, so definitely need to include it.

Also, I think we need to list `versioneer.py` there too.

If you want to include the `docs` dir, that should be included recursively. I suggest following what is listed in https://github.com/NSLS-II/scientific-python-cookiecutter/blob/main/%7B%7B%20cookiecutter.repo_name%20%7D%7D/MANIFEST.in with corresponding modifications as needed.

Reply:
Do you also want to include the `doc` dir? I think we should rename it to `docs` to be consistent with the other repos so that the copied workflows would work without many modifications in the future.